client/blocks/subscription-length-picker/docs/example.jsx

```diff
@@ -0,0 +1,46 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { getPlan } from 'lib/plans';
+import { PLAN_BUSINESS, PLAN_BUSINESS_2_YEARS } from 'lib/plans/constants';
+import { SubscriptionLengthPicker } from 'blocks/subscription-length-picker';
+import PropTypes from 'prop-types';
+
+const productsWithPrices = [
+	{
+		planSlug: PLAN_BUSINESS,
+		plan: getPlan( PLAN_BUSINESS ),
+		priceFull: 200,
+		priceMonthly: 200 / 24,
```

Comment:
What happens when we modify prices? We should only use prices from the endpoint.

Reply:
If this is only used for display purposes please note this in code. Folks have a tendency to copy pasta from examples.